### PR TITLE
Icu 9881 remove use of rose form textarea from admin UI

### PR DESCRIPTION
**Pull request by NatashaKab**
:tickets: [Jira ticket](https://hashicorp.atlassian.net/browse/ICU-9881)

## Description
remove use of rose form textarea from admin UI

This pull request will unblock the pull request to [retire rose textarea.](https://github.com/hashicorp/boundary-ui/pull/1735)

### CODE REVIEW

File: ui/admin/app/components/form/auth-method/oidc/index.hbs [hunk 1]
```diff
@@ -573,15 +573,20 @@
           {{#each @model.idp_ca_certs as |cert i|}}
             <body.row as |row|>
               <row.cell>
-                <form.textarea
+                <Hds::Form::Textarea::Field
```
Comment (ZedLi) [inline ui/admin/app/components/form/auth-method/oidc/index.hbs:576]: I believe this is also optional

File: ui/admin/app/components/form/credential-store/vault/index.hbs [hunk 2]
```diff
@@ -220,47 +219,49 @@
     {{/if}}
   </form.checkbox>
 
-  <form.textarea
-    @name='client_certificate'
-    @type='text'
+  <Hds::Form::Textarea::Field
```
Comment (ZedLi) [inline ui/admin/app/components/form/credential-store/vault/index.hbs:222]: Any reason we didn't mark any of these text areas as optional? I believe these are all optional

File: ui/admin/app/components/form/host-set/azure/index.hbs [hunk 3]
```diff
@@ -161,6 +160,26 @@
...
+    {{#if @model.errors.filter_string}}
+      <F.Error as |E|>
+        {{#each @model.errors.filter_string as |error|}}
+          <E.Message>{{error.message}}</E.Message>
+        {{/each}}
+      </F.Error>
+    {{/if}}
+  </Hds::Form::Textarea::Field>
+
   <form.textarea
```
Comment (ZedLi) [inline ui/admin/app/components/form/host-set/azure/index.hbs:183]: Looks like this didn't get deleted?

File: ui/admin/app/components/form/host-set/azure/index.hbs [hunk 4]
```diff
@@ -161,6 +160,26 @@
     </:body>
   </Rose::Form::Fieldset>
 
+  <Hds::Form::Textarea::Field
```
Comment (ZedLi) [inline ui/admin/app/components/form/host-set/azure/index.hbs:163]: I think this is also optional

**Review by ZedLi — CHANGES_REQUESTED** [review]

**Review by calcaide — CHANGES_REQUESTED** [review]
Terrific work @NatashaKab !!  those new text areas looking great!!

Almost done!! I +1 @ZedLi comment about `isOptional` and `isRequired` and the missing one within azure host-sets (left specific comment).
Appreciate the hard work!!

**Review by DhariniJeeva — APPROVED** [review]
Great job! LGTM!

**Review by DhariniJeeva — CHANGES_REQUESTED** [review]
We found that some new textarea fields aren't disabled in edit mode.

**Review by DhariniJeeva — APPROVED** [review]

**Review by ZedLi — APPROVED** [review]
Looks great!

**Review by calcaide — APPROVED** [review]
Let's gooo @NatashaKab !! great job :partying_face:

**Pull request merged**